components/tidb_query/src/expr/builtin_json.rs

```diff
@@ -204,6 +218,115 @@ mod tests {
     use crate::expr::{EvalContext, Expression};
     use tipb::ScalarFuncSig;
 
+    #[test]
+    fn test_json_length() {
+        let cases = vec![
+            (None, None, None),
+            (Some("null"), None, None),
```

Comment:
hi, here are some corner cases we need to test. and current implementation seems can not pass

```rust
// when the second argument is null, the result should always be null
(Some(r#"{}"#), Some(Datum::Null), None)
// only one argument, and is string "null", result should be 1
(Some("null"), None, Some(1)),
```

the related SQL:

```msyql
mysql> select json_length("{}", null);
+-------------------------+
| json_length("{}", null) |
+-------------------------+
|                    NULL |
+-------------------------+
1 row in set (0.00 sec)


mysql> select json_length("null");
+---------------------+
| json_length("null") |
+---------------------+
|                   1 |
+---------------------+
1 row in set (0.00 sec)

```

Reply:
Thank you. Fixed